docs/mobile-apps/automated-testing/ipa-files.md

```diff
@@ -121,3 +121,20 @@ If your organization requires specific entitlements, Sauce Labs gives you the op
 - `com.apple.developer.associated-domains`
 - `com.apple.security.application-groups`
 - `keychain-access-groups`
+
+
+## Common Errors
+
+#### Unable to Verify App
+
+If you are facing the issue where the app crashes with a red screen and an "Unable to Verify App" popup:
+<img src={useBaseUrl('img/mobile-apps/verify-app-error.png')} alt="Mobile app settings navigation" width="350"/>
+
+ that means your proxy might be blocking Apple's signature check for installing custom enterprise apps on iOS. Apple has recently started rolling out a new signature verification and PPQS check for new prov. profiles and our new accounts. During installation, Apple sends an initial API request to verify the signature of the app.
```

Comment:
Suggestion to reorg this content to explain the cause and error first, then provide the workaround.

```suggestion
Apple has a signature verification and PPQS check for new provisioning profiles and new accounts. During installation, Apple sends an API request to verify the signature of the app.

If the app crashes with a red screen and an "Unable to Verify App" error appears (as shown below), your proxy might be blocking Apple's signature check for installing custom enterprise apps on iOS.

<img src={useBaseUrl('img/mobile-apps/verify-app-error.png')} alt="Mobile app settings navigation" width="350"/>
```